### ISSUE TITLE
[FR] Maximum dimensions in terms of height and width, along with a factor for scaling

### ISSUE THREAD

**Issue by spiritualgeek** [pashashocky/obsidian-note-gallery#32]
This is indeed a promising start. However, its practical application needs the incorporation of two features:

1. The capability to define maximum dimensions in terms of height and width
2. The provision to establish a scaling or zoom factor, facilitating the reduction in size of all components, including headings

Thanks

**Comment by rudyon** [comment]
I agree with this. The text is super small and unreadable on my screen. A scale/zoom factor is needed for it to be usable for me.

**Comment by spiritualgeek** [comment]
No longer using due to activity on the plugin seeming to have ceased. Closing the request
